File: proxy-loaned-book-service/src/main/java/com/ironhack/proxyloanedbookservice/controller/dto/LoanedDTO.java
package com.ironhack.proxyloanedbookservice.controller.dto;

public class LoanedDTO {
    private Long ean;
    private Long userId;

    public LoanedDTO() {
    }

    public LoanedDTO(Long ean, Long userId) {
        this.ean = ean;
        this.userId = userId;
    }

    public Long getEan() {
        return ean;
    }

    public void setEan(Long ean) {
        this.ean = ean;
    }

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }
}

File: Final-Project-Front/src/app/models/LoanedDTO.ts
export class LoanedDTO{

  constructor(
    private _ean: number,
    private _userId:number | null
  ){}


  get ean(): number {
    return this._ean;
  }

  set ean(value: number) {
    this._ean = value;
  }

  get userId(): number | null{
    return this._userId;
  }

  set userId(value: number | null) {
    this._userId = value;
  }
}
